bin/db_patcher.js

```diff
@@ -3,211 +3,29 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var path = require('path')
-var fs = require('fs')
 var mysql = require('mysql')
-var P = require('../promise.js')
 var options = require('../config')
```

Comment:
Consider naming this `config = require('../config')` and keeping it around, rather than replacing it when you overwrite the `options` variable below.